etna/datasets/tsdataset.py

```diff
@@ -1111,21 +1113,29 @@ def get_level_dataset(self, target_level: str) -> "TSDataset":
         if target_level_index > current_level_index:
             raise ValueError("Target level should be higher in the hierarchy than the current level of dataframe!")
 
+        target_names = tuple(get_target_with_quantiles(columns=self.columns))
```

Comment:
Don't we want to make a function, that does this logic?

Reply:
You mean create method which return dataframe with target, quantiles and target components?

Reply:
Create property `target_quantiles` in dataset